client/src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import {  // only use for react-router-dom v6 and above
```

Comment:
for the React code I would advise having someone who knows some react give you feedback. I can read it and it looks good to me, but I did very little React in my coding days :|